.circleci/config.yml

```diff
@@ -77,12 +77,19 @@ jobs:
             git checkout master
             CURR_MBT_VERSION=$(cat ./VERSION)
             echo "Current release: ${CURR_MBT_VERSION}"
-            if [[-z "$MBT_VERSION_MANUAL"]]; then 
-              echo "Using manual MBT version ${MBT_VERSION_MANUAL}"
-              MBT_VERSION=$MBT_VERSION_MANUAL
-            else
-              MBT_VERSION=$(echo $CURR_MBT_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g')
-            fi
+            MBT_VERSION=$(echo $CURR_MBT_VERSION |awk -F. -v MAJOR_VAR="$MAJOR" -v MINOR_VAR="$MINOR" -v PATCH_VAR="$PATCH" '{
+                                                    if (MAJOR_VAR == "true"){
```

Comment:
Do we want to allow an option not to bump the version? (in case it was manually bumped)